develop/schemadoc/Protocol/Protocol.Params.Param.Display.RTDisplay-onAppLevel.md

```diff
@@ -16,7 +16,13 @@ Specifies whether this parameter needs to be accessible on application level.
 
 ## Remarks
 
-Set this attribute to true if the parameter needs to be accessible on application level, i.e. outside the protocol. This can for instance be in a visual overview, Automation script, or GQI query.
+Set this attribute to true if the parameter needs to be accessible on application level, i.e. outside the protocol. This can for instance be in a visual overview, Automation script, or GQI query. Note that this attribute is not known by DataMiner; it is only used by and for the protocol validator.
+
+> [!NOTE]
+> This attribute **should no longer be used**, as there is a better alternative, i.e. the Validator Suppression feature. This feature has the same possibilities, but with the following advantages:
```

Comment:
Good point, I actually thought about it but didn't fine anything about it currently on docs.

We have a generic page about the validator here: https://docs.dataminer.services/develop/TOOLS/DIS/ToolWindows/DisValidatorToolWindow.html

But it doesn't say anything about the Validator suppression feature at the moment.
Not sure if adding a link to that page on the "Validator" word is better than nothing or not. At least, people would know what we are referring to when mentionining the validator but they still wouldn't know what the suppression feature it about.
Ideally, we would need to extend the DIS Validator page with more information of the various features it has but yeah, that's no longer a quick win :)

Reply:
Indeed, that goes beyond the scope of this PR, but it would be good if we could make sure this information also becomes available. @MichielOda, is expanding the validator documentation something you could take care of (in a separate PR)?

Reply:
Info added in following PR #3865